README.md

```diff
@@ -115,6 +115,7 @@ In order to deal with data that come from different sources (e.g. ROS, EtherCAT,
 Data from the black box can be retrieved through a Zyre-based query interface that listens to JSON messages and responds back with JSON messages. The following message types can be sent to the query interface:
 * variable query: returns a list of all variables logged on a particular black box (schema defined in [docs/messages/black-box-variable-query-schema.json](docs/messages/black-box-variable-query-schema.json))
 * data query: returns data corresponding to a set of variables in a given time interval (schema defined in [docs/messages/black-box-data-query-schema.json](docs/messages/black-box-data-query-schema.json))
+* latest data query: returns the latest datum corresponding to a set of variables
```

Comment:
We need to include a schema for this message, just like for the other message types.

Reply:
Resolved [here](https://github.com/ropod-project/black-box/pull/2/commits/66486c0b4fd273d89b7b242160daa2fde277bea8).